.stylelintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+	"rules": {
+		"block-no-empty": null,
+		"color-no-invalid-hex": true,
+		"comment-empty-line-before": [
+			"always",
+			{
+				"ignore": ["stylelint-commands", "after-comment"]
+			}
+		],
+		"declaration-colon-space-after": "always",
+		"indentation": [
+			2,
+			{
+				"except": ["value"]
+			}
+		],
+		"max-empty-lines": 4,
+		"rule-empty-line-before": [
+			"never-multi-line",
+			{
+				"except": ["first-nested"],
+				"ignore": ["after-comment", "inside-block"]
+			}
+		],
+		"unit-allowed-list": ["em", "rem", "%", "s", "px", "vh", "deg", "pt"]
+	}
+}
```

Comment:
the indentation of this file is odd, can you ensure it uses 4 spaces please?

Reply:
sure

Reply:
It's already 4 spaces when viewed in my IDE. I don't know why indentation changes when pushed to GitHub.

Reply:
it's because it's using tabs instead of spaces, I suggest enabling the "show white space option in your editor" so you can see this. Can you change from tabs to spaces please? We do this for consistency with the rest of the codebase which uses only spaces.

Reply:
Got it